lib/client.ml

```diff
@@ -1,17 +1,16 @@
 open Lwt_unix
-open Util
```

Comment:
Feel free to change the `.ocamlformat` directly in this branch. I just copy/past the one from Deku for homogeneity.